src/entities/VerifiablePresentation.entity.ts

```diff
@@ -179,13 +179,25 @@ async function getPresentationByIdentifier(holderDID: string, presentationIdenti
 
 }
 
-
+async function deleteAllPresentationsWithHolderDID(holderDID: string): Promise<void> {
```

Comment:
It looks like this function is never called, is that intentional?

Reply:
Added more fixes, including this.